src/commands/plugins/update.test.js

```diff
@@ -1,6 +1,9 @@
 // @flow
 
 import PluginsUpdate from './update'
+import fs from 'fs-extra'
+fs.readFileSync = jest.fn()
+fs.readFileSync.mockReturnValue(' { "skipAnalytics": true }')
```

Comment:
seems like this is testing the implementation of the config module. I think it would be better to do `new Config({skipAnalytics: true})`